_docs/setup/kubernetes/sidecar-injection.md

```diff
@@ -160,7 +166,7 @@ kubectl apply -f install/kubernetes/istio.yaml
 
 Webhooks requires a signed cert/key pair. Use `install/kubernetes/webhook-create-signed-cert.sh` to generate 
 a cert/key pair signed by the Kubernetes' CA. The resulting cert/key file is stored as a Kubernetes 
-secret for the sidecar injector webhook to consume.
+secret for the sidecar injector webhook to consume. Before running the script, please install cfssl. On MAC OS X, you can use 'brew install cfssl'.
```

Comment:
nack. I've updated the scripts to use openssl. see https://github.com/istio/istio/pull/2911